spec/models/repository/git_spec.rb

```diff
@@ -14,12 +14,12 @@
     end
 
     it 'should create a bulk job on a queue' do
-      expect { repository.remote_send('clone') }.
+      expect { repository.fetch('clone') }.
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.